src/providers/web_workers/worker/auth.ts

```diff
@@ -58,6 +58,18 @@ export class WebWorkerFirebaseAuth extends AuthBackend {
   getAuth(): FirebaseAuthData {
     return this._fbRef.getAuth();
   }
+  
+  createUser(creds: FirebaseCredentials): Promise<FirebaseAuthData> {
+    return new Promise<FirebaseAuthData>((resolve, reject) => {
+      this._fbRef.createUser(creds, (err, authData) => {
```

Comment:
While this is true, the typings don't reflect that so I have to wrap it for the time being. I'll see if I can get typings updated so we can simply return from the method.